source/security/index.markdown

```diff
@@ -1,11 +1,81 @@
 ---
 title: "Security"
-description: "Information about disclosing security vulnerabilities in Home Assistant."
+description: "Home Assistant takes its security seriously. This page contains information about how we handle security issues, how to report them, and information on past security issues."
 ---
 
-If you think that you have found a security vulnerability in Home Assistant, please disclose it to us via our security e-mail address at [security@home-assistant.io](mailto:security@home-assistant.io).
+Home Assistant takes its security seriously. We will do everything in our power to ensure that our users are safe.
 
-Please do not make vulnerabilities public without notifying us and giving us at least 3 days to respond.
+This page is intended to provide information about how we handle security issues and information about security issues we have [handled in the past](#past-advisories).
 
-If you are going to write about Home Assistant's security, please get in touch, so we can make sure that all claims are correct.
-  
+## Reporting a vulnerability
+
+So, you have found a security vulnerability in Home Assistant? Please, be sure to **responsibly disclose** it to us by [reporting a vulnerability using GitHub's Security Advisory](https://github.com/home-assistant/core/security/advisories/new).
+
+**DO NOT MAKE A PUBLIC ISSUES FOR SECURITY VULNERABILITIES!**
+
+We are mostly interested in reports by actual Home Assistant users that are familiar with the platform, but all high quality contributions are welcome. Please do your best to describe a clear and realistic impact for your report.
+
+For the sake of the security of our users, please 🙏 do not make vulnerabilities public without notifying us and giving us at least 90 days to release a fixed version. We will do our best to respond to your report within 7 days and also to keep you informed of the progress of our efforts to resolve the issue, but understand that Home Assistant, like many open source projects, is relying heavily on volunteers that aren't full-time resources. We may not be able to respond as quickly as you would like due to other responsibilities.
+
+If you are going to write about Home Assistant’s security, please get in touch, so we can ensure that all claims are correct.
```

Comment:
How do you want people to get in touch?

Reply:
Contact information is in the footer of every page. Will point to that.